cargo/cargo_build_script_runner/lib.rs

```diff
@@ -144,7 +144,11 @@ impl BuildScriptOutput {
         v.iter()
             .filter_map(|x| {
                 if let BuildScriptOutput::DepEnv(env) = x {
-                    Some(format!("{}{}", prefix, Self::redact_exec_root(env, exec_root)))
+                    Some(format!(
```

Comment:
> For the future, if you're `rustfmt`ing things and also changing them, it can be nice to do those in two separate commits, to make it easier to review what _you_ have changed, vs what rustfmt has changed :)

Fortunately, this is the only thing that changed as a result of running `rustfmt`. But I can definitely keep the changes separated in future PRs 👍